package.json

```diff
@@ -17,7 +17,8 @@
     "django:serve": "python manage.py runserver 0.0.0.0:${PORT:-8000}",
     "worker:serve": "python manage.py rqworker default short",
     "scheduler:serve": "python manage.py rqscheduler --queue short",
-    "rq:serve": "run-p worker:serve scheduler:serve",
+    "redis:clear": "redis-cli -h redis FLUSHALL",
```

Comment:
@wlonk This is breaking when I run `yarn serve` outside of docker, since the `redis` hostname can't be resolved.

Reply:
@wlonk is there a reason you didn't go with the Python script approach I suggested in https://github.com/oddbird/MetaDeploy/pull/23#issuecomment-663083020 ?  That would ensure it's connecting to the same redis configured in REDIS_URL, and would also avoid needing to have redis-cli installed.

Reply:
Because this seemed a smaller quicker change, that's all.

Under what circumstances are you running `yarn serve` outside of Docker? We're running it on Heroku with an explicit call to Daphne, not `yarn serve`, so I was unaware that was even a use-case.

Reply:
I don't use Docker for local development of MetaDeploy. I find it cumbersome.

Reply:
We have both ways documented in https://github.com/SFDO-Tooling/MetaDeploy/blob/main/CONTRIBUTING.rst